<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1724</width>
    <height>806</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>781</width>
      <height>46</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="bttn_START">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_STOP">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_RESET">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bttn_BACK_MAIN">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Main Menu</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>901</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Time Elasped:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_TIMER"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Weight:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcd_WEIGHT"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>430</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>430</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>100%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="sld_DUTYCYCLE">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>400</y>
      <width>321</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn_RESUME">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>380</y>
      <width>183</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Resume</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn_PAUSE">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>430</y>
      <width>183</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QLabel" name="camera_image">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>401</width>
      <height>289</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>72</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IMAGE</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_FORCE" native="true">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>70</y>
      <width>411</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_POSITION" native="true">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>70</y>
      <width>411</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
   <widget class="PlotWidget" name="graph_STRAIN" native="true">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>70</y>
      <width>411</width>
      <height>289</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>301</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Platen to Platen Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_DISTANCE_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Top Platen to Sample Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_DISTANCE_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>710</y>
      <width>411</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sample to Bottom Platen Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_to_PLATEN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_to_SAMPLE">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>620</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_DISTANCE_4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>521</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Sample  Top to Original Sample Top Distance:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_SAMPLE_TOP_to_ORIGINAL_SAMPLE_TOP">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>520</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_PLATEN_DISTANCE_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>610</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Original Sample Top to Original Bottom Platen:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ORIGINALSAMPLE_to_BOTTOM_PLATEN">
    <property name="geometry">
     <rect>
      <x>1130</x>
      <y>610</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lb_SAMPLE_to_BOTTOM_PLATEN">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>710</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="bttn_CHANGE_DIRECTION">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>480</y>
      <width>183</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Change 
Direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_FPS">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>700</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QWidget</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
